### PR TITLE
feature/5624 add preview for 2 links

### PR DESCRIPTION
**Pull request by ndegendogo**
## Description

This PR implements preview for the two following links:
- Statistics info, FAQ (AppStrings.Statistics.Info.faqLink)
- Info for risk card with low-risk encounters, FAQ (AppStrings.Home.riskEncounterLowFAQLink)

The link preview feature was requested at #2179.

Also I did a small cleanup in the ExposureDetectionViewModel.
[This function](https://github.com/corona-warn-app/cwa-app-ios/blob/db3bfc81668bc266b15913260733c7be66f6f5de/src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift#L610) `explanationSection` is never called with a positive `numberOfExposures`; 
so I could remove the parameter and the related code.

**Note:** PR #3877 covers mocking of the links during unit tests. I recommend to merge #3877 before this PR.
*Note 2: Originally I had planned to cover also the FAQ at Error reports (AppStrings.ErrorReport.faqURL); but the accessibility identifier did not work as expected, and the robots caught it.*

## Link to Jira

Jira: Internal Tracking ID: [EXPOSUREAPP-5624](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5624) 
github: #2179

## Screenshots

Simulator Screen Shot - iPhone 8 - 2021-11-27 at 11 14 38

Simulator Screen Shot - iPhone 8 - 2021-11-27 at 11 14 56

Simulator Screen Shot - iPhone 8 - 2021-11-27 at 11 15 33

Simulator Screen Shot - iPhone 8 - 2021-11-27 at 11 15 42

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift [hunk 1]
```diff
@@ -618,14 +618,6 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
 				accessibilityIdentifier: accessibilityIdentifier
 			)
 		]
-		if numberOfExposures > 0 {
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift:621]: Why does this get removed? Seems much needed?

**Review by marcussc — CHANGES_REQUESTED** [review]

**Review by pascal-brause — COMMENTED** [review]
Same question like @marcussc: Removing the check for `numberOfExposures > 0` changes the logic. Why removing it?

**Comment by ndegendogo** [comment]
@marcussc @pascal-brause 
Well ... actually I was looking for an execution path to that code, because I wanted to test the behaviour of the link ... 
but I couldn't find one; so I concluded that it is dead code (maybe forgotten to cleanup after a refactoring ... ).
But of course, I may be wrong.
And even if not: if you want to keep this code, e.g. for future use, I don't insist to delete it, I can bring it back.

*Still I would like to understand it ... I still don't see the execution path ...*
*There are two functions `explanationSection(...)` in this class, and both functions are private - so we can limit our analysis to this class, right?*
*[This one](https://github.com/corona-warn-app/cwa-app-ios/blob/6efafb8b22d78db8abb6b4fd5fecef695717703c/src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift#L610) is called [from the inactiveModel](https://github.com/corona-warn-app/cwa-app-ios/blob/6efafb8b22d78db8abb6b4fd5fecef695717703c/src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift#L340) and [from the failureModel](https://github.com/corona-warn-app/cwa-app-ios/blob/6efafb8b22d78db8abb6b4fd5fecef695717703c/src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift#L358) and both don't have a parameter `numberOfExposures`, so they get the default value -1 and don't hit that condition.*
*The [lowRiskModel](https://github.com/corona-warn-app/cwa-app-ios/blob/6efafb8b22d78db8abb6b4fd5fecef695717703c/src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift#L408) has such a parameter, but it calls [the other function](https://github.com/corona-warn-app/cwa-app-ios/blob/6efafb8b22d78db8abb6b4fd5fecef695717703c/src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift#L636).*
*And the highRiskModel calls a separate function [highRiskExplanationSection](https://github.com/corona-warn-app/cwa-app-ios/blob/6efafb8b22d78db8abb6b4fd5fecef695717703c/src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift#L460).*

**Review by marcussc — APPROVED** [review]

**Review by pascal-brause — APPROVED** [review]
After having a small pair-review with @marcussc, we agree 100% to your solution. We tried a little bit around and came to the same opinion like you.
Nice finding and thanks for your contribution. 🎉

**Pull request merged**